.circleci/config.yml

```diff
@@ -57,9 +57,9 @@ jobs:
       - restore_cache: *restore-yarn-cache
       - restore_cache: *restore-node-modules-cache
       - run: yarn --version
-      - run: make test-ci-coverage
+      - run: node Makefile.js testCiCoverage
       # Builds babel-standalone with the regular Babel config
-      - run: IS_PUBLISH=true make build
+      - run: IS_PUBLISH=true node Makefile.js build
```

Comment:
Please use `yarn node` instead of `node`, so that we won't have to update it when migrating to yarn 2.